src/scripts/components/Tooltip.jsx

```diff
@@ -8,6 +8,12 @@ import { TimelineMax } from 'gsap/TweenMax';
 import { UtilitySystem } from '../UtilitySystem';
 
 class Tooltip extends React.Component {
+  constructor(props) {
```

Comment:
Just throwing it out for discussion, in rhinofront we have avoided constructors and gone with es7 property initializer syntax.  I'm not sure if rhinostyle is set to work with those or not, but if so I wonder if we should try to keep it consistent?

Reply:
@juanfabrega Good point, we could either move to `componentDidMount` or if it needed it before then we'd have to bring back the constructor, which I'm fine with.

Reply:
Yup. This was more a comment on future changes to other components. This one doesn't need it.